components/desktop/pulseaudio/patches/08-oss4.patch

```diff
@@ -1,225 +1,6 @@
---- pulseaudio-6.0/src/modules/oss/module-oss.c.~1~	2015-02-12 17:10:35.000000000 +0300
```

Comment:
Right, I rebased on oi/hipster before working on the component, not sure what happened.